doc/MIGRATION-v6.md

```diff
@@ -248,6 +248,8 @@ We've removed setter methods `setIntVal`, `setKind`, `setStrVal` from the class.
 
 The only exception to the above is `patch(PatchContext, item)` - it is valid to be called after withName. 
 
+CustomResourceList was deprecated - use DefaultKubernetesResourceList instead.
```

Comment:
This is supposed to be a list entry, right?
```suggestion
- CustomResourceList was deprecated - use DefaultKubernetesResourceList instead.
```

Reply:
@rohanKanojia I added your change along with making the previous line continue from the preceding bullet point.